doc/colvars-refman-main.tex

```diff
@@ -4181,6 +4181,7 @@ \subsection*{Overview}
 \cvsubsec{Moving frame of reference.}{sec:colvar_atom_groups_ref_frame}
 
 The following options define an automatic calculation of an optimal translation (\texttt{centerReference}) or optimal rotation (\texttt{rotateReference}), that superimposes the positions of this group to a provided set of reference coordinates.
+Alternately, \texttt{centerToOrigin} applies a translation to place the geometric center of the group at (0, 0, 0).
```

Comment:
It seems here that `centerToOrigin` is just a short-hand for `centerReference` without the need to specify a `dummyAtom` group with coordinates on the origin. So perhaps why not present it like that instead of calling it an "alternative"?